integration_tests/test_argv_01.py

```diff
@@ -0,0 +1,12 @@
+from ltypes import i32
+from sys import argv
+
+def test():
+    exe: str = argv()[0]
```

Comment:
I think this has to be:
```suggestion
    exe: str = argv[0]
```

Reply:
Exactly, but global `list` doesn't work yet.